cmd/boltdb/main.go

```diff
@@ -1,10 +1,15 @@
 package main
```

Comment:
@esimonov I believe we need to create other cmd app specific for plugins instead of refactoring boltdb app default app